crates/oct-ctl/Cargo.toml

```diff
@@ -4,4 +4,10 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
-actix-web = { workspace = true}
\ No newline at end of file
+actix-web = { workspace = true}
+futures = "0.3.31"
+log = "0.4"
+serde = {  workspace = true, features = ["derive"] }
+serde_json = { version = "1.0", workspace = true }
+env_logger = { workspace = true }
+reqwest = { workspace = true }
```

Comment:
Here we shouldn't use versions of the dependencies, they are defined in the root `Cargo.toml`